src/service/share-service.js

```diff
@@ -125,9 +126,9 @@ var ShareService = exports.ShareService = Montage.specialize({
     save: {
         value: function (shareObject, isServiceEnabled) {
             var saveSharePromise;
-            delete shareObject.permissions.user;
-            delete shareObject.permissions.group;
-            delete shareObject.permissions.others;
+            // delete shareObject.permissions.user;
```

Comment:
I thought someone added it for purpose and it might be required for other type of shares other than smb shares.
Removed as per your request.